src/daemon.py

```diff
@@ -208,10 +225,12 @@ def process_transactions(chain_name: str) -> None:
         solver_slippage_db_connection,
     ) = initialize_connections()
     rt = RawTokenImbalances(web3, chain_name)
+    api_prices = [
+        CoingeckoPriceProvider(),
+        DunePriceProvider(),
+        MoralisPriceProvider(),
```

Comment:
I would like to see a provider for native prices from the auction as well. Should be relatively easy to add since it can be fetched from the solver competition endpoint.